plan/property_cols_prune.go

```diff
@@ -40,6 +40,42 @@ func (p *LogicalSelection) preparePossibleProperties() (result [][]*expression.C
 	return p.children[0].preparePossibleProperties()
 }
 
+func (p *LogicalSort) preparePossibleProperties() (result [][]*expression.Column) {
+	p.children[0].preparePossibleProperties()
+	result = make([][]*expression.Column, 0, 1)
```

Comment:
I think we can extract a function for the following.

Reply:
+1, maybe name it to "getSortPropertyFromGbyCols" ?

Reply:
maybe we can merge this two operator in the future.

Reply:
Done